App/Services/RdpConnector.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Coder.Desktop.App.Services;
+
+public struct RdpCredentials(string username, string password)
+{
+    public readonly string Username = username;
+    public readonly string Password = password;
+}
+
+public interface IRdpConnector : IAsyncDisposable
+{
+    public const int DefaultPort = 3389;
+
+    public Task WriteCredentials(string fqdn, RdpCredentials credentials, CancellationToken ct = default);
```

Comment:
If this doesn't need to be async then it should probably just be synchronous for now. If we do need to change it to be async then we can do it pretty easily in the future.